test/pg_handler/tests/ssl_test.go

```diff
@@ -145,6 +145,49 @@ func TestSSL(t *testing.T) {
 				SSLRootCertType: Undefined,
 			},
 		},
+		{
+			TestDefinition: TestDefinition{
+				Description:         "server_tls, sslmode=verify-ca, sslrootcert=valid, sslkey=malformed, sslcert=malformed",
+				ShouldPass:          false,
+				CmdOutput:           StringPointer("psql: FATAL:  tls: failed to find any PEM data in certificate input"),
+			},
+			AbstractConfiguration: AbstractConfiguration{
+				ListenerType:      TCP,
+				ServerTLSType:     TLS,
+				SSLModeType:       VerifyCA,
+				SSLRootCertType:   Valid,
+				SSLPrivateKeyType: PrivateKeyMalformed,
+				SSLPublicCertType: PublicCertMalformed,
+			},
+		},
+		{
+			TestDefinition: TestDefinition{
+				Description:         "server_tls, sslmode=verify-ca, sslrootcert=valid, sslkey=valid, sslcert=valid",
+				ShouldPass:          true,
+			},
+			AbstractConfiguration: AbstractConfiguration{
+				ListenerType:      TCP,
+				ServerTLSType:     TLS,
+				SSLModeType:       VerifyCA,
+				SSLRootCertType:   Valid,
+				SSLPrivateKeyType: PrivateKeyValid,
+				SSLPublicCertType: PublicCertValid,
+			},
+		},
+		{
+			TestDefinition: TestDefinition{
```

Comment:
This is probably my own misunderstanding, but it's confusing that the cert here is _not_ signed by the CA, the sslModeType is `VerifyCA`, and it's expected to pass.  Can we add a note explaining this.

Reply:
VerifyCA is to do with `sslmode`.
It means the client wants to validate the certificate of the server against a certificate the client will provide.

This is distinct from the validation of client private key and certificate. Notably, the client can not configure this. It's at the server's discretion. E.g. mysql checks to ensure the client certificate has been signed by the root CA on the server.

I personally don't think here is the place to add this information.